catalog/spatial/country/spatial-country-local-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+           xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0
+           http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+           http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0
+           http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd">
+
+    <bean id="localCountryCodeConverter"
+          class="org.codice.ddf.spatial.country.converter.impl.LocalCountryCodeConverter">
+        <cm:managed-properties
+                persistent-id="org.codice.ddf.spatial.country.converter.impl.LocalCountryCodeConverter"
+                update-strategy="container-managed"/>
+        <property name="fipsToIso3MappingFile" value="${ddf.home}/etc/fipsToIso.properties"/>
```

Comment:
Since this is a properties file, we need to update the migration code to copy the file over, https://github.com/codice/ddf/blob/master/platform/platform-migratable/src/main/java/org/codice/ddf/platform/migratable/impl/PlatformMigratable.java if we move this service to platform. If we keep it under catalog, let me know and we can see where this can be added.